packages/react-components/src/menu/src/Menu.css

```diff
@@ -150,7 +150,8 @@ IMPORTANT: The Menu component hardcoded a few CSS values, do not remove or renam
 .o-ui-menu-section-title {
     cursor: default;
     display: flex;
-    font-weight: 500;
+    font-weight: 400;
+    font-variation-settings: "wght" 500;
```

Comment:
I haven't confirm but my feeling was the even the font-size wasn't the same between Listbox and Menu sections

Reply:
they both share a `font-size: var(--o-ui-global-type-scale-8);` which is fine since section titles are all caps.